database/bb_sp_db.py

```diff
@@ -0,0 +1,67 @@
+import os
+from os import path
+import logging
+import sqlalchemy as sql
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base 
+from sqlalchemy import Column, Float, Integer, String, MetaData
+
+
+# Getting the parent directory of this file. That will function as the project home.
+PROJECT_HOME = path.dirname(path.dirname(path.abspath(__file__)))
+
+# Logging
+LOGGING_CONFIG = path.join(PROJECT_HOME, 'config/logging.conf')
+
+# Offline database
+DATABASE_PATH = path.join(PROJECT_HOME, 'data/billboard_spotify.db')
+SQLITE_ENGINE = 'sqlite:////{}'.format(DATABASE_PATH)
+
+# RDS database 
+CONN_TYPE = "mysql+pymysql"
+USER = os.environ.get("MYSQL_USER")
+PASSWORD = os.environ.get("MYSQL_PASSWORD")
+HOST = os.environ.get("MYSQL_HOST")
+PORT = os.environ.get("MYSQL_PORT")
+DATABASE_NAME = os.environ.get("DATABASE_NAME")
+MYSQL_ENGINE = "{}://{}:{}@{}:{}/{}".format(CONN_TYPE, USER, PASSWORD, HOST, PORT, DATABASE_NAME)
+
+
+Base = declarative_base()
+
+class Billboard_Spotify(Base):
+
```

Comment:
White space here probably isn't necessary. See the comments I've made regarding white space for more details. I am not going to go through every single file and mark whitespace because it's entirely your choice whether to keep it!